### PR TITLE
Migrate alert message

### PR DESCRIPTION
**Pull request by masone**

### CODE REVIEW

File: src/stories/alertMessage.stories.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  alignCenter: true,
+  fullWidth: true,
+}
+
+export const FullWidth = Template.bind({})
+FullWidth.args = {
+  alertName: "Full width",
+  alignCenter: false,
+  fullWidth: true,
+}
```
Comment (Marine-Berthier) [inline src/stories/alertMessage.stories.js:61]: It looks like some stories are missing : `success` and the ones `with icons`. Regarding the `type`, do you think a dropdown within the control pannel could be enough ? Something like the one to select `alignment` and` position` for the [`Tooltip`](https://github.com/carforyou/carforyou-components-pkg/pull/286/files)

**Comment by Marine-Berthier** [comment]
@masone I worked directly on your branch and so pushed my changes directly here : was it the right/best way to do it ?

**Comment by masone** [comment]
I can't approve my own PR, so you'll have to do it :)
Then, feel free to merge!

**Review by Marine-Berthier — APPROVED** [review]

**Comment by masone** [comment]
thanks for the support Marine

**Pull request merged**